### PR TITLE
Added two new JavaScript resources

### PR DESCRIPTION
**Pull request by codeguage-code**
## Resource name and link

[Comprehensive JavaScript Course - CodeGuage](https://www.codeguage.com/courses/js/)

## Short description

A detailed course on JavaScript, covering numerous aspects of the language, with 15 units (modules), almost 80 chapters, 25 quizzes, and over 50 exercises.

## Why do you think this is a good resource?

First of all, the resource is absolutely free of cost. There's even no need to sign up to access the content. In addition to this, unlike many free resources out there, the chapters go in extensive detail and offer top-notch, high-quality content, both easy to read and easy to understand.

### CODE REVIEW

**Review by AvidCoder101 — APPROVED** [review]
LGTM!

File: JavaScript.md [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 - [Exploring JS: JavaScript books for programmers](http://exploringjs.com/) by Dr. Axel Rauschmeyer: free online books that delve deep into the Javascript language - always up-to-date with new books on the latest Ecmascript features
 
+- [A Comprehensive Course on JavaScript - CodeGuage](http://www.codeguage.com/courses/js/): An absolutely free and comprehensive course of JavaScript, starting with the very fundamentals of the language before moving over to consider some of its most intricate aspects, such as closures, prototypes, HTML DOM, CSSOM, events, and so on, sprinkled with numerous quizzes and exercises for you to practice your skills as you learn.
+
+- [100+ JavaScript Questions - CodeGuage](http://www.codeguage.com/questions/js/): A whole bundle of questions, along with their answers, regarding the foundational aspects of JavaScript.
+
 - [A brief history of JavaScript](https://medium.com/@benastontweet/lesson-1a-the-history-of-javascript-8c1ce3bffb17) by Ben Aston: It is a medium article describing and throwing light upon the History of JavaScript.
```
Comment (CBID2) [inline JavaScript.md:19]:
```suggestion
- [A brief history of JavaScript](https://medium.com/@benastontweet/lesson-1a-the-history-of-javascript-8c1ce3bffb17) by Ben Aston: It is a Medium article describing and throwing light upon the History of JavaScript.
```

File: JavaScript.md [hunk 2]
```diff
@@ -12,6 +12,10 @@
...
 
 - [FreeCodeCamp Beginning JavaScript](https://www.freecodecamp.org/): Learn JavaScript with an online community, examples, and tons of more advanced JavaScript topics when you are done. Certificate available too. All free.
```
Comment (CBID2) [inline JavaScript.md:21]:
```suggestion
- [FreeCodeCamp Beginning JavaScript](https://www.freecodecamp.org/): Learn JavaScript with an online community, examples, and tons of more advanced JavaScript topics. Best part is: you get a certificate and it's all free!
```

File: JavaScript.md [hunk 3]
```diff
@@ -12,6 +12,10 @@
 
 - [Exploring JS: JavaScript books for programmers](http://exploringjs.com/) by Dr. Axel Rauschmeyer: free online books that delve deep into the Javascript language - always up-to-date with new books on the latest Ecmascript features
```
Comment (CBID2) [inline JavaScript.md:13]:
```suggestion
- [Exploring JS: JavaScript books for programmers](http://exploringjs.com/) by Dr. Axel Rauschmeyer: This free online book series devels deep into the Javascript language. The best part: It's always up-to-date with new books on the latest Ecmascript features!
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @codeguage-code! 😄 I added some suggestions to make your descriptions a bit clearer. I hope you find them helpful! 😄

**Pull request merged**